modules/spark/src/test/scala/org/apache/ignite/spark/AbstractDataFrameSpec.scala

```diff
@@ -129,6 +129,27 @@ abstract class AbstractDataFrameSpec extends FunSpec with Matchers with BeforeAn
         cache.put("key3", Employee(3, "Arnold Schwarzenegger", 27, 1000))
     }
 
+    def createMetaTestTable(client: Ignite, cacheName: String): Unit = {
```

Comment:
Let's move these methods to the `IgniteDataFrameSchemaSpec` because it used only there.

Reply:
Moved